index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Chingu</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" type="image/png" href="img/logo.png">
+    
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+        crossorigin="anonymous">
+
+</head>
+
+<body>
+
+    <header class="header">
+        <!-- start code here -->
+    </header>
+    <section class="carrosel">
+
+    </section>
+    <section class="about">
+
+      </section>
+      <section class="container-fluid portfolio">
+          <div class="row">
+           <div class="col-md-8 col-md-offset-2 text-center portfolio-1">
+               <a href="https://www.donut.ai">
+                   <img src="img/Donut.png" alt="Portfolio 1" class="img-responsive">
+                 </a>                
+                  <p>Portfolio 1</p>
```

Comment:
whose portfolio it is? I think we should link to our team project.

Perhaps we can create other projects after this landing page 🤔

Reply:
We can discuss it